lang/de/settings.php

```diff
@@ -128,6 +128,11 @@
         'three_month' => '3 Monate (90 Tage)',
         'tile_description' => 'Regelt systemweitere Einstellungen wie Logo, Wartungs-Banner und Seitengrößen.',
         'title' => 'Konfiguration',
+        'toast_lifetime' => [
+            'custom' => 'Benutzerdefiniert',
+            'description' => 'Zeit in Sekunden, die eine Benachrichtigung angezeigt wird',
```

Comment:
Consider changing to 'Zeit in Sekunden, für die eine Benachrichtigung angezeigt wird'